csharp/sdk/4.0/models.cs

```diff
@@ -2608,24 +2608,9 @@ public enum InvestigativeContentType
 
 public class JsonBi : SdkModel
 {
-  public JsonBiBigQueryMetadata big_query_metadata { get; set; } = null;
-  public JsonBiFields fields { get; set; } = null;
-  /// <summary>Pivots (read-only)</summary>
-  public JsonBiPivots[] pivots { get; set; } = null;
-  /// <summary>If the query has subtotals (read-only)</summary>
-  public bool has_subtotals { get; set; }
-  /// <summary>If the query has totals (read-only)</summary>
-  public bool has_totals { get; set; }
-  /// <summary>If the query results hit the maximum column limit and additional columns were truncated (read-only)</summary>
-  public string columns_truncated { get; set; } = "";
-  /// <summary>Filter expression applied to the query results (read-only)</summary>
-  public string filter_expression { get; set; } = "";
-  /// <summary>Filters applied to the query results (read-only)</summary>
-  public StringDictionary<string> filters { get; set; } = null;
-  /// <summary>Raw sql query. Null if user does not have permission to view sql (read-only)</summary>
-  public string sql { get; set; } = "";
+  public JsonBiMetadata metadata { get; set; } = null;
```

Comment:
I guess we'll find out if customers complain about this breaking change.

Reply:
Yeah i noticed this. We need to discuss this internally later on how preventing or mitigating breaking API changes.